source/_docs/guides/drupal8-commandline.md

```diff
@@ -0,0 +1,376 @@
+
+We don't need the `--sync-content` flag when going to the Live environment because that environment already has our canonical database.
+
+#### Import the configuration on the Live environment
+
+`terminus drush $TERMINUS_SITE.live -- config-import -y`
+
+Once this command completes you will be able to refresh the Live environment in your browser and see the development footer.
+
+## Pulling content changes down from the Live environment to the Dev environment
+
+In the lifecycle of managing a site, you can expect content editors to add new material to the Live environment.
+You may need this fresh command in your Development environment.
+
+#### Make content on the Live environment
+
+Since you are signed into the Live environment, you can manually make content by browsing to `/node/add`.
+We can also use a Drush command from `devel_generate` module to create some number of nodes.
+
+`terminus drush $TERMINUS_SITE.live  -- generate-content 25`
+
+Including "25" at the end of that command gives us 25 nodes.
+
+#### Bring down the database and media files from the Liv Environment to the Dev environment
+
+`terminus env:clone-content  $TERMINUS_SITE.live dev`
+
+We are cloning content from `live` to `dev`.
+
+## Combining code changes and content changes in the Test Environment before deploying to Live
+
+#### Make another configuration change on the Dev environment
+
+`terminus drush  $TERMINUS_SITE.dev -- views-enable glossary`
+
+That command will enable the glossary View that displays content by month.
+You can see it by pointing your browser to `/glossary` on your Dev site.
+
+#### Export the configuration change in the Dev environment
+
+`terminus drush  $TERMINUS_SITE.dev -- config-export -y`
+
+#### Commit the configuration change in the Dev environment
+
+`terminus env:commit  $TERMINUS_SITE.dev --message="Enabling glossary View"`
+
+#### Check the Test environment
+
+Before we deploy our configuration change enabling the glossary View to the Test environment, let's first see what the Test environment looks like.
+Visit `/glossary` and `/admin/content` in your Test environment.
+You should see a 404 message for the glossary page and the administrative content list should not contain the articles and pages that were made on live.
+Once we deploy our code in the next step, we should see something different on both URLs.
+
+#### Deploy the configuration change to Test
+
+These are the same commands we ran above to deploy to Test and then import configuration from files to the database.
+
+`terminus env:deploy $TERMINUS_SITE.test --sync-content --updatedb --cc  --note="Deploying glossary View"`
+
+`terminus drush  $TERMINUS_SITE.test  -- config-import -y`
+
+#### Check the Test environment again
+
+Sign into the Test site again.
+Copying down the Live database likely signed you out of the Test environment.
+
+`terminus drush $TERMINUS_SITE.test -- user-login`
+
+Visit `/glossary` and `/admin/content` again.
```

Comment:
`drush user-login` does allow you to add a target URL parameter, so you could update to use this feature here if desired.